src/services/service.js

```diff
@@ -296,6 +296,31 @@ class Service {
     })
 
   }
+
+  handleInspect(projectId, dbType, col){
+    return new Promise((resolve, reject) => {
+      this.client.getJSON(`/v1/api/config/inspect/${projectId}/${dbType}/${col}`)
+      .then(({status, data}) => {
+        if(status !== 200){
+          reject("Internal server error")
+          return
+        }
+        resolve(data)
```

Comment:
Directly resolve data.schema here and use schema in the DbSchema page instead of `res`